refactron/cicd/github_actions.py

```
@@ -0,0 +1,244 @@
+errors = summary.get('errors', 0)
+warnings = summary.get('warnings', 0)
+
+print(f"📊 Analysis Summary:")
+print(f"  Critical: {{critical}}")
+print(f"  Errors: {{errors}}")
+print(f"  Warnings: {{warnings}}")
+print(f"  Total: {{summary.get('total_issues', 0)}}")
+
+# Quality gate checks
+fail = False
+if {str(quality_gate.get('fail_on_critical', True))} and critical > {quality_gate.get('max_critical', 0)}:
```

Ruff (0.14.13)

Use explicit conversion flag

Replace with conversion flag

(RUF010)